frontend/src/e2e-test/specs/0_citizen/citizen-discussion-surveys.spec.ts

```diff
@@ -0,0 +1,409 @@
+// SPDX-FileCopyrightText: 2017-2024 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import { DevCalendarEventTime, DevPerson } from 'e2e-test/generated/api-types'
+import FiniteDateRange from 'lib-common/finite-date-range'
+import HelsinkiDateTime from 'lib-common/helsinki-date-time'
+import LocalDate from 'lib-common/local-date'
+import LocalTime from 'lib-common/local-time'
+
+import {
+  createDaycarePlacementFixture,
+  Fixture,
+  testAdult,
+  testCareArea,
+  testChild,
+  testDaycare,
+  uuidv4
+} from '../../dev-api/fixtures'
+import {
+  createChildren,
+  createDaycarePlacements,
+  resetServiceState
+} from '../../generated/api-clients'
+import CitizenCalendarPage from '../../pages/citizen/citizen-calendar'
+import { DiscussionSurveyModal } from '../../pages/citizen/citizen-discussion-surveys'
+import CitizenHeader, { EnvType } from '../../pages/citizen/citizen-header'
+import { waitUntilEqual } from '../../utils'
+import { Modal, Page } from '../../utils/page'
+import { enduserLogin } from '../../utils/user'
+
+const e: EnvType[] = ['desktop', 'mobile']
+
+let page: Page
+let header: CitizenHeader
+let calendarPage: CitizenCalendarPage
+let children: DevPerson[]
+const today = LocalDate.of(2022, 1, 3)
+
+const groupEventId = uuidv4()
+const unitEventId = uuidv4()
+const individualEventId = uuidv4()
+const reservationId = uuidv4()
+const groupReservationId = uuidv4()
+const restrictedEventId = uuidv4()
+const restrictedEventTimeId = uuidv4()
+const noncancellableEventTimeId = uuidv4()
+
+let reservationData: DevCalendarEventTime
+
+beforeEach(async () => {
+  await resetServiceState()
+  await Fixture.careArea(testCareArea).save()
+  await Fixture.daycare({ ...testDaycare, areaId: testCareArea.id }).save()
+
+  await Fixture.person(testChild).saveChild({ updateMockVtj: true })
+
+  await Fixture.person(testAdult).saveAdult({
+    updateMockVtjWithDependants: [testChild]
+  })
+
+  children = [testChild]
+
+  const cs = children.map(({ id }) => ({
+    id,
+    additionalInfo: '',
+    allergies: '',
+    diet: '',
+    dietId: null,
+    mealTextureId: null,
+    languageAtHome: '',
+    languageAtHomeDetails: '',
+    medication: ''
+  }))
+
+  await createChildren({ body: cs })
```

Comment:
Tämän sijaan riittää yksinkertaisesti yksi `Fixture.family`. Se luo henkilöt ja lisää huoltajuustiedon mock-VTJ:hin.
```suggestion
  await Fixture.family({
    guardian: testAdult,
    children: [testChild]
  }).save()
```

Reply:
Muutettu käyttämään `Fixture.family` funktiota.